src/main/scala/org/broadinstitute/dsde/agora/server/webservice/QueryHandler.scala

```diff
@@ -14,29 +12,31 @@ import spray.httpx.SprayJsonSupport._
 import spray.routing.RequestContext
 
 /**
- * Actor responsible for querying the methods repository.
+ * QueryHandler is an actor that receives web service requests and calls AgoraBusiness logic.
+ * It then handles the returns from the business layer and completes the request. It is responsible for querying the
+ * methods repository for methods and method configurations.
  */
 class QueryHandler(authorizationProvider: AuthorizationProvider) extends Actor {
   implicit val system = context.system
 
-  val agoraBusiness = new AgoraBusiness()
+  val agoraBusiness = new AgoraBusiness(authorizationProvider)
 
   def receive = {
     case QuerySingle(requestContext: RequestContext,
-                     namespace: String,
-                     name: String,
-                     snapshotId: Int,
-                     entityType: Seq[AgoraEntityType.EntityType],
-                     username: String,
-                     onlyPayload: Boolean) =>
+    namespace: String,
```

Comment:
Why was the indenting removed here. I thought we decided we liked that? See every case class under Model. But this is a case match of course. Does intelliJ complain or something?